src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java

```diff
@@ -889,6 +893,83 @@ private JButton getCleanTableButton() {
 		return resetButton;
 	}
 
+	private JButton getValidateButton() {
+		validateButton = new JButton(MessageBundle.getMessage("angal.inventory.validate.btn"));
+		validateButton.setMnemonic(MessageBundle.getMnemonic("angal.inventory.validate.btn.key"));
+		if (inventory == null) {
+			validateButton.setEnabled(false);
+		}
+		validateButton.addActionListener(actionEvent -> {
+				if (inventory == null) {
+					MessageDialog.info(null, "angal.inventory.inventorymustsavebeforevalidate.msg");
+					return;
+				}
+				if (inventory != null) {
+					if (inventory.getStatus().equals(InventoryStatus.validated.toString())) {
+						MessageDialog.info(null, "angal.inventory.inventoryalreadyvalidate.msg");
+						return;
+					}
```

Comment:
This looks uncorrect: validate operation should always be possible because, despite the inventory itself is not changed, might be changed the theoretical quantities in lots from other users operations at a later time

Reply:
remember that anyway "Validation is performed every time before CONFIRMATION"

Reply:
The Aims of this issue is to implement the action validate inventory and this action imply create also the charge and discharge movement when the information is correct to do its:
1- enter the inventory information and inventory row
2- save those information by click on save Button
3- validate by click on validate Button
    validatation include:
  1- check if all informations to make this action (validation) is correct 
  2- if yes make the charge movement if realQty > theoQty in inventory row or the inventory row has new lot, make discharge movement  else.
   3- if all going fine return true otherwise return the message error

NB: this action is make one time for one inventory

Reply:
nope... the movements are created within the Confirm operation OP-1305
Validation is just for validation